test/log.ts

```diff
@@ -633,6 +646,41 @@ describe('Log', () => {
       assert.ok(message.startsWith('hello world'));
       assert.ok(message.length < maxSize + entryMetaMaxLength);
     });
+
+    it('should not contin duplicate or illegal fields to be truncated and defaults should present', async () => {
+      assert.ok(log.jsonFieldsToTruncate.length > 1);
```

Comment:
nit: would it be possible to assert the exact length instead? it should be a length of the predefined + 1, right?

Reply:
We can - the only inconvenience will be broken tests every time we add values to `log.jsonFieldsToTruncate`... 
It would make sense to check exact amount of fields if by adding extra field we would have some extra steps to perform (e.g. adding some handler per entry), but in this case adding an entry is kinda no brainer.

Reply:
yes, this could serve a kind of control to make people review the test if they make such change. leave it to your considerations.

Reply:
Thanks - I will leave it as is for now, but will keep it in mind if we will ever need this as a way to control additions